lib/kubeclient/common.rb

```diff
@@ -241,6 +241,14 @@ def define_entity_methods
         define_singleton_method("patch_#{entity.method_names[0]}") do |name, patch, namespace = nil|
           patch_entity(entity.resource_name, name, patch, namespace)
         end
+
+        define_singleton_method("json_patch_#{entity.method_names[0]}") do |name, patch, namespace = nil|
+          json_patch_entity(entity.resource_name, name, patch, namespace)
+        end
+
+        define_singleton_method("merge_patch_#{entity.method_names[0]}") do |name, patch, namespace = nil|
+          merge_patch_entity(entity.resource_name, name, patch, namespace)
+        end
```

Comment:
Travis failed on rubocop.

```
lib/kubeclient/common.rb:207:7: C: Metrics/BlockLength: Block has too many lines. [30/25]
```
It's this block, of all the `define_singleton_method`s.  It's perfectly readable.
Can you find a way to suppress this?

```
lib/kubeclient/common.rb:245:101: C: Metrics/LineLength: Line is too long. [105/100]
        define_singleton_method("json_patch_#{entity.method_names[0]}") do |name, patch, namespace = nil|
                                                                                                    ^^^^^
lib/kubeclient/common.rb:249:101: C: Metrics/LineLength: Line is too long. [106/100]
        define_singleton_method("merge_patch_#{entity.method_names[0]}") do |name, patch, namespace = nil|
                                                                                                    ^^^^^^
```
You can copy the ugly ` \ ` style from `get_` and `delete_` above :-/
Could also try creating shorter vars for `entity.method_names[1]` and `entity.method_names[0]`.

Or apply `rubocop:disable` comments around this whole function.  These only add noise to the code :-(, but this particular block has been annoying contributors for some time...

Reply:
Usually I'm not a fan of going around rubocop but I think with regards to the block length it makes sense to do so.

Concerning line length, 100 seems a bit draconian to me. I think something closer to 120 still maintains readability and gets us that much more space. For now, I'll adopt the `/` convention to stay in line with the rest of the code